Examples/Algorithms/TrackFinding/src/TrackParamsEstimationAlgorithm.cpp

```diff
@@ -23,18 +22,28 @@ ActsExamples::TrackParamsEstimationAlgorithm::TrackParamsEstimationAlgorithm(
     Acts::Logging::Level lvl)
     : ActsExamples::BareAlgorithm("TrackParamsEstimationAlgorithm", lvl),
       m_cfg(std::move(cfg)) {
+  // Either seeds directly or proto tracks + space points
   if (m_cfg.inputSeeds.empty()) {
-    throw std::invalid_argument("Missing input seeds collection");
+    if (m_cfg.inputSpacePoints.empty()) {
+      throw std::invalid_argument("Missing space point input collections");
```

Comment:
Really small detail but maybe the error should say "Missing seed or space point input collections" ?

Reply:
Thats a good point. Fixed.